lib/bank_accounts/checking.rb

```diff
@@ -0,0 +1,55 @@
+require 'pry'
+
+module BankAccounts
+  class CheckingAccount < Account
+
+    attr_reader :balance, :checks_used
+
+    def initialize(id, initial_balance)
+      # inherit from Account class
+      super
+
+      # start with 0 checks used
+      @checks_used = 0
+    end
+
+    def withdraw(amount)
+      #incur a fee of $1 with each withdrawal
+      amount += 1
```

Comment:
Transaction fee in the instructions is $2. Always good to double check details before committing to master.